ros/src/tl_detector/tl_detector.py

```diff
@@ -180,21 +286,30 @@ def get_light_state(self, light):
             int: ID of traffic light color (specified in styx_msgs/TrafficLight)
 
         """
-
+        labels = list(enumerate(['Red', 'Yellow', 'Green', 'None', 'None']))
```

Comment:
Why there are 2 'None' labels? Can you add a comment describing what is the difference between them?

Reply:
This is mostly for debugger use, I am using enum to reverse retrieve names for color labels. 0-red, 1-yellow, 2-green, 4-none. I had to insert something for 3, event it's not used, because TL can't be 3.

Reply:
I think this explanation should be in a comment.